docs/docs/guides/web3_migration_guide/providers_migration_guide.md

```diff
@@ -4,78 +4,11 @@ sidebar_position: 2
 sidebar_label: web3.providers
 ---
 
-There are multiple ways to set the provider.
```

Comment:
Are following steps different from 1.x and changed in 4.x?

Reply:
Actually, this describes how to set the provider and this does not have any breaking changes (as far as I know). For that, I moved it to the Guides...